src/scopes/goals/index.js

```diff
@@ -6,14 +6,15 @@ import { withTopics, withoutTopics } from './topics';
 import { withReasons, withoutReasons } from './reasons';
 import { withRecipientId } from './recipientId';
 import { withRegion, withoutRegion } from './region';
+import { withRoles, withoutRoles } from './role';
 import { withGrantNumber, withoutGrantNumber } from './grantNumber';
 
 export const topicToQuery = {
   createDate: {
     bef: (query) => beforeCreateDate(query),
     aft: (query) => afterCreateDate(query),
     win: (query) => withinCreateDate(query),
-    is: (query) => withinCreateDate(query),
+    in: (query) => withinCreateDate(query),
```

Comment:
Was this change intentional?

Reply:
Yeah, all the other date scopes were "in", and the frontend was sending an "in"